nat/rules.go

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2019 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package nat
+
+import (
+	"strings"
+
+	"github.com/mysteriumnetwork/node/config"
+	"github.com/rs/zerolog/log"
+)
+
+// ProtectNetworksRule creates a NAT rule to block packets to protected networks if set.
+func ProtectNetworksRule(sourceSubnet, targetIP string) *Rule {
+	s := config.GetString(config.FlagFirewallProtectedNetworks)
```

Comment:
Try to not depend on global state here. This method can be pure function which creates nat rule from given params. It would be more clear if you check if FlagFirewallProtectedNetworks is set before calling this method and pass servers string directly.

Reply:
The point of extracting this to a separate function was to encapsulate the logic and improve readability of the `openvpn/manager.Serve`. The boilerplate part of this is string parsing so if it is left out, there's no point to extract it anymore. Passing server list as a single comma separated string to the function does not seem right as it says little about the contract: it would make more sense to pass already parsed IPs.